akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -327,6 +327,76 @@ object Flow {
    */
```

Comment:
Please add this docs note here too https://github.com/akka/akka/pull/22876/files#r115651026